integration-tests/pom.xml

```diff
@@ -1,4 +1,4 @@
-<!-- Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+<!-- Copyright (c)  2020, Oracle Corporation and/or its affiliates.
```

Comment:
Need to undo this change to copyright statement.

Reply:
Oops.